app.rb

```diff
@@ -57,7 +78,53 @@ def add_a_music_album
     puts 'Add a new music album'
   end
 
+  def add_game(game)
+    @games << game
+  end
+
+  def find_or_create_author(firstname, lastname)
+    existing_author = @authors.find { |author| author.firstname == firstname && author.lastname == lastname }
+    if existing_author
+      existing_author
+    else
+      new_author = Author.new(firstname, lastname)
+      @authors << new_author
+      new_author
+    end
+  end
+
   def add_a_game
     puts 'Add a new game'
+    print 'Enter the game title: '
+    game_title = gets.chomp
+
+    print 'Enter the number of players: '
+    num_players = gets.chomp.to_i
+
+    print 'Enter the last played date (YYYY-MM-DD): '
+    last_played_date = gets.chomp
+
+    new_game = Game.new(num_players, last_played_date)
+    new_game.title = game_title
+    
+    puts 'Add a new author'
+    print 'Enter the author\'s first name: '
+    firstname = gets.chomp
```

Comment:
Here you should modify the `Item` class first to add an association relationship so you can avoid a lot of unneeded, please reach out to me so we can discuss it.